src/Autofac.Integration.SignalR/Autofac.Integration.SignalR.nuspec

```diff
@@ -14,7 +14,7 @@
     <iconUrl>https://cloud.githubusercontent.com/assets/1156571/13684110/16b8f152-e6bf-11e5-84ae-22c66c6d351a.png</iconUrl>
     <releaseNotes>Release notes are at https://github.com/autofac/Autofac.SignalR/releases</releaseNotes>
     <dependencies>
-      <dependency id="Autofac" version="[3.5.0,5.0.0)" />
+      <dependency id="Autofac" version="[3.5.0,5.0.0]" />
```

Comment:
I think this should be [5.0.0, 6.0.0) now.

Because of the targeting change, this package is now compatible only with minimum 5.0.0, up until the next breaking change (i.e. 6.0.0).